github/github.go

```diff
@@ -200,6 +201,17 @@ func (c *Client) NewUploadRequest(urlStr string, reader io.Reader, size int64, m
 	return req, nil
 }
 
+// sanitizeErr removes client_id and client_secret variables from URL in
+// net/http error, making it safe to expose.
+func sanitizeErr(err error) error {
+	if err == nil {
+		return nil
+	}
+	re := regexp.MustCompile(`client_(id|secret)=([^&|\s|:]*)(&)?`)
```

Comment:
@parkr You're right, my mistake. The commit I just pushed places the regex outside the func so that it's only compiled once at compile time.